components/util/constants.tsx

```diff
@@ -169,3 +169,9 @@ export const sswColors = {
   "#AAAAAA": "bg-ssw-gray-light",
   "#797979": "bg-ssw-gray",
 };
+
+export const CustomClasses = {
```

Comment:
```suggestion
export const customClasses = {
```

Maybe also make it specific to margin i.e. `customMarginClasses`

Reply:
I'm thinking of having some more classes here in the future to have more customization, currently, it only contains margin but we might be adding some padding as well